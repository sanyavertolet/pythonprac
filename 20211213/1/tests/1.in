[22, 58, 95, 33, 47, 11, 76, 38, 70, 84, 40, 35, 56, 28, 13, 23]
