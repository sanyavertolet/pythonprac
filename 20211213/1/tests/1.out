[11, 13, 22, 23, 28, 33, 35, 38, 40, 47, 56, 58, 70, 76, 84, 95]
